config/ct_windows.sys.config

```diff
@@ -0,0 +1,69 @@
+[
+    {dcos_rest, [
+      {ip, {127, 0, 0, 1}},
+      {port, 62080}
+    ]},
+
+    {dcos_l4lb, [
+        {enable_networking, false}
+    ]},
+
+    {dcos_dns, [
+        {handler_limit, 16},
+        {mesos_resolvers, [{{127, 0, 0, 1}, 62053}]},
+        {upstream_resolvers, [
+            {{8, 8, 8, 8}, 53},
+            {{8, 8, 4, 4}, 53},
+            {{1, 1, 1, 1}, 53}
+        ]},
+        {exhibitor_url, "http://master.mesos:8181/exhibitor/v1/cluster/status"},
+        {udp_port, 8053},
+        {tcp_port, 8053}
+    ]},
+
+    {erldns,[
+        {storage, [
+            {type, erldns_storage_json},    %% erldns_storage_json | erldns_storage_mnesia
+            {dir, "db"},
+            {dbname, undefined},  %% name of the db
+            {user, undefined},    %% for future db types
+            {pass, undefined},    %% for future db types
+            {host, undefined},    %% for future db types
+            {port, undefined}     %% for future db types
+        ]},
+        {servers, [[
+            {name, inet_localhost_1},
+            {address, "127.0.0.1"},
+            {port, 62053},
+            {family, inet},
+            {processes, 1}
```

Comment:
could you please explain this changes?

```
--- config/ct.sys.config	2017-09-18 17:45:20.000000000 -0700
+++ config/ct_windows.sys.config	2018-03-28 09:26:21.000000000 -0700
@@ -36,7 +36,7 @@
             {address, "127.0.0.1"},
             {port, 62053},
             {family, inet},
-            {processes, 10}
+            {processes, 1}
         ]]},
         {use_root_hints, false},
         {catch_exceptions, false},
```

Reply:
if you really need these changes I'd prefer to change `ct.sys.config` in such a way:

```
    {erldns,[
        {servers, [[
            {name, inet_localhost_1},
            {address, "127.0.0.1"},
            {port, 62053},
            {family, inet},
            {processes, 1}
        ]]},
        {use_root_hints, false},
        {catch_exceptions, false},
        {zones, "data/zones.json"},
        {pools, [
            {tcp_worker_pool, erldns_worker, [
                {size, 10},
                {max_overflow, 20}
            ]}
        ]}
    ]},
```

Reply:
I can't really explain it since I haven't looked into the actual root cause for it.
Running the unit tests with the default config almost all of them fail with the following:
```** State machine <0.3975.0> terminating
** Last event = {cast,init}
** When server state  = {execute,
                            {state,
                                {dcos_dns_tcp_handler,<0.3968.0>},
                                undefined,
                                <<84,11,1,32,0,1,0,0,0,0,0,1,6,109,97,115,116,
                                  101,114,5,109,101,115,111,115,0,0,1,0,1,0,0,
                                  41,16,0,0,0,0,0,0,12,0,10,0,8,134,173,66,9,
                                  247,97,112,175>>,
                                [],60111872,undefined}}
** Reason for termination = error:{badmatch,{error,econnrefused}}
** Callback mode = handle_event_function
** Stacktrace =
**  [{dcos_dns_handler_fsm,do_tcp_resolve,3,
                           [{file,"c:/dcos-net/_build/windows+test/lib/dcos_dns/src/dcos_dns_handler_fsm.erl"},
                            {line,263}]},
     {dcos_dns_handler_fsm,resolve,3,
                           [{file,"c:/dcos-net/_build/windows+test/lib/dcos_dns/src/dcos_dns_handler_fsm.erl"},
                            {line,227}]},
     {dcos_dns_handler_fsm,start_workers,2,
                           [{file,"c:/dcos-net/_build/windows+test/lib/dcos_dns/src/dcos_dns_handler_fsm.erl"},
                            {line,354}]},
     {dcos_dns_handler_fsm,execute,1,
                           [{file,"c:/dcos-net/_build/windows+test/lib/dcos_dns/src/dcos_dns_handler_fsm.erl"},
                            {line,126}]},
     {gen_statem,call_state_function,5,[{file,"gen_statem.erl"},{line,1242}]},
     {gen_statem,loop_event,6,[{file,"gen_statem.erl"},{line,1012}]},
     {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]```

I'll look into the root cause :).

Reply:
Ok, I think I found the issue:
```
http://erlang.org/pipermail/erlang-bugs/2012-October/003108.html
https://github.com/erlang/otp/blob/76c4821cff1e91c26dd2f812fc8cb40aa7d48e57/erts/emulator/drivers/common/inet_drv.c#L6228
```

Reply:
It's sad that BEAM doesn't supported SO_REUSEADDR on Windows. So if my [suggestion](https://github.com/dcos/dcos-net/pull/38#discussion_r177816258) works for you could you please make changes in `ct.sys.config` and remove `ct_windows.sys.config`?

Reply:
Sure, I will update it.

Reply:
Updated.